src/coreclr/tools/r2rtest/CompilerRunner.cs

```diff
@@ -224,7 +224,8 @@ protected virtual ProcessParameters ExecutionProcess(IEnumerable<string> modules
             }
 
             // TODO: support for tier jitting - for now we just turn it off as it may distort the JIT statistics
-            processParameters.EnvironmentOverrides["COMPLUS_TieredCompilation"] = "0";
+            processParameters.EnvironmentOverrides["DOTNET_TieredCompilation"] = "0";
+            processParameters.EnvironmentOverrides["COMPlus_TieredCompilation"] = "0";
```

Comment:
Why both?

Reply:
Does this run against pre-.NET 6 runtimes?

Reply:
@BruceForstall asked for both when we are resetting variables https://github.com/dotnet/runtime/pull/76997#discussion_r995150098.